core/process/locales.loader.php

```diff
@@ -86,14 +175,14 @@
 
 foreach ($pokemons->pokemon as $pokeid => $pokemon) {
 	// Merge name and description from translation files
-	$pokemon->name 				= $pokemon_trans->pokemon->$pokeid->name;
-	$pokemon->description 		= $pokemon_trans->pokemon->$pokeid->description;
+	$pokemon->name = $pokemon_trans->pokemon->$pokeid->name;
+	$pokemon->description = $pokemon_trans->pokemon->$pokeid->description;
 
 	// Replace quick and charge move with translation
-	$quick_move 				= $pokemon->quick_move;
-	$pokemon->quick_move 		= $pokemon_trans->quick_moves->$quick_move;
-	$charge_move 				= $pokemon->charge_move;
-	$pokemon->charge_move 		= $pokemon_trans->charge_moves->$charge_move;
```

Comment:
Yup it was scrutinizer that asked me to change it, but as I put on discord scrutinizer patchs told me one thing and issues told me another about my closure functions so maybe I should not pay too much attention to it.